src/main/resources/static/views/welcome/welcome.html

```diff
@@ -29,6 +32,13 @@ <h4 class="h4TileHeader"><i class="icon-people"></i> Project Members</h4>
                 </div>
             </div>
             <h4 class="h4TileHeader"><i class="icon-rocket"></i> My Projects</h4>
+            <div class="alert alert-danger" ng-cloak ng-show="isProfOrCoordinator">

```

Comment:
maybe this would look better if we move it on top like we did for students.

Reply:
It’ll be in the same position as he student alert once it’s the prof/coordinator that’s logged in